rest-mvc/src/main/java/com/intuit/tank/rest/mvc/TankAPIApplication.java

```diff
@@ -15,7 +15,7 @@
 import org.springframework.boot.web.servlet.support.SpringBootServletInitializer;
 import org.springframework.context.ConfigurableApplicationContext;
 
-@OpenAPIDefinition(servers = {@Server(url = "/", description = "Default Server URL")})
+@OpenAPIDefinition(servers = {@Server(url = "/api", description = "Default Server URL")})
```

Comment:
@Zakaria-Kofiro If we're adding this to add an `/api` prefix to all endpoints, I suggest we stick with the previous `/` for convention